contracts/contracts/chains/ethereum/EthereumMessengers.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "OpenZeppelin/openzeppelin-contracts@4.8.0/contracts/access/Ownable.sol";
+
+import "../../../interfaces/IMessenger.sol";
+import "../../RestrictedCalls.sol";
+import "../../Resolver.sol";
+
+contract EthereumL1Messenger is IMessenger, RestrictedCalls {
+    function callAllowed(address, address) external view returns (bool) {
+        // The call must be delivered by the Resolver
+        // Since we are on the same chain as the Resolver
+        // we should never see an execution of this function.
+        revert("Unexpected state.");
+    }
+
+    // Simply forward the message to the target on L1
+    /* solhint-disable avoid-tx-origin */
+    function sendMessage(address target, bytes calldata message)
```

Comment:
Do we need `    /* solhint-disable avoid-tx-origin */` here even though we don't mention `tx.origin` explicitly?

Reply:
You are right, forgot to remove this line.